packages/react/src/lib/shouldHandleOnKeys.ts

```diff
@@ -3,18 +3,26 @@ import * as keyboardKey from 'keyboard-key'
 import * as _ from 'lodash'
 import * as React from 'react'
 
+const shouldHandleModifier = (modifierValue: boolean, combinationValue?: boolean) => {
+  if (typeof combinationValue === 'undefined') {
+    return true
+  }
+
+  return modifierValue === combinationValue
+}
+
 const shouldHandleOnKeys = (
```

Comment:
nit: may suggest to rename this method to `matchesAnyCombination` - this will better reflect its intent

Reply:
I was renamed to `shouldHandleOnKeys` here https://github.com/stardust-ui/react/pull/1421 and most of the people approved. Frankly, I like it more as we know what is the expected answer we want to receive - if we need to handle keydown events. `matchesAnyKeyCombination ` - doesn't answer this question for me.